File: 20190611-kotlin-fizzbuzz/src/main/kotlin/com/katas/FizzBuzzer.kt
package com.katas

class FizzBuzzer {
    fun fizzBuzz(number: Int): String {
        return when {
            number.isMultipleOf(3) && number.isMultipleOf(5) -> "FizzBuzz"
            number.isMultipleOf(5) -> "Buzz"
            number.isMultipleOf(3) -> "Fizz"
            else -> number.toString()
        }
    }
}

fun Int.isMultipleOf(number: Int): Boolean {
    return this % number == 0
}

File: 20200511-python-fizzbuzz/fizzbuzzer.py
class FizzBuzzer(object):
    def fizz_buzz(self, number):
        if number % 3 == 0 and number % 5 == 0:
            return "FizzBuzz"
        if number % 5 == 0:
            return "Buzz"
        if number % 3 == 0:
            return "Fizz"
        return str(number)
